lte/gateway/c/core/oai/tasks/amf/nas_proc.cpp

```diff
@@ -412,10 +413,55 @@ int amf_nas_proc_auth_param_res(
   OAILOG_FUNC_RETURN(LOG_NAS_AMF, rc);
 }
 
+static int subs_auth_retry(zloop_t* loop, int timer_id, void* arg) {
+  amf_ue_ngap_id_t ue_id                 = 0;
+  amf_context_t* amf_ctxt_p              = NULL;
+  int amf_cause                          = -1;
+  nas5g_auth_info_proc_t* auth_info_proc = NULL;
+  int rc                                 = RETURNerror;
+  ue_m5gmm_context_s* ue_mm_context      = NULL;
+  if (!amf_pop_timer_arg(timer_id, &ue_id)) {
+    OAILOG_WARNING(
+        LOG_AMF_APP,
+        "auth_retry_timer: Invalid Timer Id expiration, Timer Id: %d \n",
+        timer_id);
+    OAILOG_FUNC_RETURN(LOG_NAS_AMF, RETURNok);
+  }
+  ue_mm_context = amf_ue_context_exists_amf_ue_ngap_id(ue_id);
+  amf_ctxt_p    = &ue_mm_context->amf_context;
+  amf_ctxt_p->auth_retry_count++;
+  OAILOG_ERROR(
```

Comment:
Why is this an error? This should be a info or a warning.

Reply:
comment addressed.